### DISCUSSION TITLE
Armor stand errors

### DISCUSSION THREAD

**Discussion by Creeperz653** [Kas-tle/java2bedrock.sh#192] [Converter Help] [answered]
@
![A25B3DBB-8187-4438-A784-C8B08EB950CF](https://user-images.githubusercontent.com/76449952/182740197-cc76aeef-236a-4ac1-8516-72e114131b16.png)

they should be the same size

**Comment by Creeperz653** [comment]
it works fine on Java the last time I checked

**Reply by Creeperz653** [reply]
The pixels are the same size in block bench

**Comment by Kas-tle** [comment]
Please attach the resource pack you are using, as well as the original java resource pack.

**Reply by Creeperz653** [reply]
https://github.com/Kas-tle/java2bedrock.sh/issues/134

**Reply by Creeperz653** [reply]
Any thing new

**Reply by Creeperz653** [reply]
@Kas-tle

**Comment by Creeperz653** [comment, marked as answer]
It seems like this is a internal bug I'm going to close this
